main.go

```diff
@@ -120,14 +122,31 @@ func readPackages() {
 
 // modified will use git to find out which folders have been changed
 func modified() {
-	cmdArgs := []string{
-		"--no-pager",
-		"-C",
-		*gitDirPtr,
-		"diff",
-		"--name-only",
-		*commitFromPtr,
-		*commitToPtr,
+
+	var cmdArgs []string
+	switch *includeTestFiles {
+	case true:
+		cmdArgs = []string{
+			"--no-pager",
+			"-C",
+			*gitDirPtr,
+			"diff",
+			"--name-only",
+			*commitFromPtr,
+			*commitToPtr,
+		}
+	default:
+		cmdArgs = []string{
+			"--no-pager",
+			"-C",
+			*gitDirPtr,
+			"diff",
+			"--",
+			`":!*test*"`,
+			"--name-only",
+			*commitFromPtr,
+			*commitToPtr,
```

Comment:
This could probably be refactored a little as both arrays are similar but it's not a major concern for me.